packages/thirdweb/src/pay/convert/cryptoToFiat.ts

```diff
@@ -0,0 +1,46 @@
+import type { Address } from "abitype";
+import type { ThirdwebClient } from "../../client/client.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import { getPayConvertCryptoToFiatEndpoint } from "../utils/definitions.js";
+
+export type ConvertCryptoToFiatParams = {
+  client: ThirdwebClient;
+  /**
+   * The contract address of the token
+   * For native token, use NATIVE_TOKEN_ADDRESS
+   */
+  fromTokenAddress: Address;
+  fromAmount: number;
+  /**
+   * The chainId that the token is deployed to
+   */
+  chainId: number;
+  /**
+   * The fiat symbol. e.g "usd"
+   */
+  to: string;
```

Comment:
can we make these consts? otherwise its hard to know what the endpoint accepts

Reply:
updated

Reply:
ah I see what you mean. Right now only "usd" is supported so will do `to: "usd"`
Once we add more currency I can update that section